BUILD.md

```diff
@@ -270,3 +270,21 @@ The ct Evm.StepN interface is used to evaluate N instructions in different EVM i
 - Differential tests; execute instruction in two VM implementations and compare state results in addition to panic.
    - FuzzDifferentialLfvmVsGeth: ```make fuzz-lfvm-diff```
    - FuzzDifferentialEvmzeroVsGeth: ```make fuzz-evmzero-diff``` (disabled, issue #549)
+
+
+## Static analysis
+
+Tosca project uses static analysis to improve code quality.
+
+### Go
+
+Currently, two different static analysis tools are used:
+- [Vet](https://pkg.go.dev/cmd/vet) Is the default go linter, bundled with the toolchain. If finds rather critical bugs that should be addressed right away. To execute it run `go vet ./go/...`. This tool is currently enabled in the CI, and pull request will be prevented from merging when checks are not satisfied.
+- [StaticCheck](https://staticcheck.dev/) Finds other [errors](https://staticcheck.dev/docs/checks/) related to styling and good practices. 
+
+Lints can be disabled as described [here](https://golangci-lint.run/usage/false-positives/). Please use `//nolint:pass` where pass is the linter pass that discovers the error. Do not ignore all passes as one line may hide more than one error.
+
+### C++
+
+There is no current static analysis for the evmzero. Integrating clang-tidy should not be too difficult. 
```

Comment:
```suggestion
There is current no static analysis for the the C++ code in this project.
```
It is not very useful for any reader to state that the integration of `clang-tidy` should not be too hard. I, as a reader, would not go and implement it, nor would I believe that it is "not too difficult", otherwise it would have been implemented.